build.proj

```diff
@@ -123,6 +123,12 @@
     </ItemGroup>
   </Target>
   
+  <PropertyGroup>
```

Comment:
Didn't you want to move this closer to where it was being used for the first time?